lib/measured/conversion_table_builder.rb

```diff
@@ -24,6 +24,8 @@ def cached?
   private
 
   def generate_table
+    validate_no_cycles
```

Comment:
Interesting. Why did you put this before generating the table? I guess because you're acting on the units themselves and not the resulting table.

Reply:
Yes, I found that in some situations, for example, with Example 1 in the issue #136, the algorithm to build the table would ignore some of the units specified in `.build`. 
In addition, the table represents a bigger graph (fully connected graph), and it would make this validation more expensive and harder to implement as I would have to validate that all cycles in the fully connected graph have a neutral net cost (e.g 1 litters -> cases -> pallets -> litters = 1).

Reply:
Great answer 👍